client-library/package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Fiberplane<info@fiberplane.com>",
   "type": "module",
   "main": "dist/index.js",
-  "version": "0.2.0",
+  "version": "0.2.1-beta.3",
```

Comment:
already published if you want to test: https://www.npmjs.com/package/@fiberplane/hono?activeTab=versions